examples/shader/post_processing.rs

```diff
@@ -42,11 +42,43 @@ fn setup(
 ) {
     let window = windows.primary_mut();
     let size = Extent3d {
-        width: window.physical_width(),
-        height: window.physical_height(),
+        width: window.width() as u32,
```

Comment:
This will actually render this at a lower resolution when the scale factor is above 1, which isn't what we want generally. Scenes should still render at the native resolution. It is _ui_ that needs to be scaled up.